app/src/main/res/layout/key_gear_item_detail_valuation_section.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/base_margin_triple">
+
+    <com.hedvig.app.ui.view.HedvigCardView
+        android:layout_width="0dp"
+        android:layout_height="100dp"
```

Comment:
dimen

Reply:
Damn, I even noted to myself that I was supposed to fix it before pushing 😞 I'll make sure to add a TODO next time 🙂